packages/ui/package.json

```diff
@@ -12,7 +12,8 @@
       "@ifixit/app": "workspace:*",
       "@ifixit/cart-sdk": "workspace:*",
       "@ifixit/icons": "workspace:*",
-      "@ifixit/newsletter-sdk": "workspace:*"
+      "@ifixit/newsletter-sdk": "workspace:*",
+      "regenerator-runtime": "*"
```

Comment:
Can `regenerator-runtime` be a `devDependency`, or does it need to be a direct one?

Reply:
You're right, it can be a `devDependency`! Fixed